docs/napari-plugin-guide.md

```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
 ```
 napari -w recOrder-napari
 ```
+## Connect to MicroManager
```

Comment:
Not a big deal, but the official spelling seems to be [Micro-Manager](https://micro-manager.org/Micro-Manager_User's_Guide).

Reply:
Fixed. Thanks!